bedrock/firefox/redirects.py

```diff
@@ -498,4 +498,7 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 1260423
     redirect(r'^firefox/choose/?$', 'firefox.new'),
+
+    # bug 1273837
+    redirect(r'^firefox(?:\/\d+\.\d+\.?(?:\d+)?\.?(?:\d+)?(?:a?)(?:\d*))?/hello/start/?$', 'firefox.hello'),
```

Comment:
I think there are one too many digit capturing groups here - the above would match `46.0.1.2` and `46.0.1.2a2`, which I don't think we need (uh, right?).

Assuming an `a` must always be followed by a number, I think you simplify slightly to:

`^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/hello/start/?$`

So, "firefox" followed by at least `#.#`, then optionally another `.#` and optionally `a#` at the end.